utils/__init__.py

```diff
@@ -264,10 +264,44 @@ def add_xmlns_sbml_attribute(sedml_filepath, sbml_filepath, output_filepath=None
     with open(output_filepath,"w") as fout:
         fout.write(sedstr)
 
+def add_xmlns_fbc_attribute(sedml_filepath, sbml_filepath, output_filepath=None):
+    '''
+    add an xmlns:sbml attribute to the sedml file that matches the sbml file
+    raise an error if the attribute is already present
+    output fixed file to output_filepath which defaults to sedml_filepath
+    '''
+    sedstr = ""
+    if output_filepath:
```

Comment:
Here it reads the data from output_filepath is it exists and ignores sedml_filepath. This seems a bit counter intuitive: for the case where the sedml_filepath and output_filepath both already exist, but are not the same file the user presumably wishes to overwrite output_filepath with the corrected version of sedml_filepath rather than read input from the output_filepath. If sedml_filepath == output_filepath then there is no problem reading input from sedml_filepath, whereas if they are different then I think output_filepath should be overwritten with the corrected version of sedml_filepath.

Reply:
I've tried to increase the clarity of the function and docstring. If there already is a temp SED-ML file (that could already contain the xlmns:sbml fix), this one should be used over the original SED-ML file. 

Is it clearer like this?